src/client/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -159,5 +159,11 @@ public partial interface IPublicClientApplication : IClientApplicationBase
             IEnumerable<string> scopes,
             string username,
             string password);
+
+        /// <summary>
+        /// Used to determine if the currently available broker is able to perform Proof-of-Possesion.
+        /// </summary>
+        /// <returns>Boolean indicating Proof-of-Possesion is supported</returns>
+        public bool IsProofOfPosessionSupportedByClient();
```

Comment:
Yes, this is a breaking change, because if someone implements our interface, you'll have broken them. The solution here is to use an extension method to add it to the interface.

Reply:
Yea, but the app builders return IPCA or ICCA, so you would have to cast to base, which is not good dev experience in my opinion - very hidden.

That's why I'm starting to like the Graph client approach more - it's a class not an interface. And it's still testable; just extend it and override. Also better for testing on our side than the static extensions that we add.

Reply:
I agree, interfaces were a bad SDK design choice, but we are stuck with them.

Reply:
Where do you suggest we add this helper @pmaytak ?

Reply:
IPCA extension method?

Reply:
Yeah, Added extension method for IPCA